### ISSUE TITLE
Media queries inlined

### ISSUE THREAD

**Issue by BroiSatse** [Mange/roadie#130]
Having example css:
```
.test {
  color: green;
}
@media screen and (max-width 480px) {
  .test {
    color: red;
  }
}
```
`test` div ends up with `styles="color:green;color:red"` effectively making it red regardless of the size screen. I am using foundation inky, so I do not have much control over the styles and I cannot easily separate mobile form desktop stylesheets. Any way around this problem?

**Comment by Mange** [comment]
That's strange. What version of css_parser are you having in your
Gemfile.lock?

Den fre 25 nov. 2016 19:38Stanislaw Klajn <notifications@github.com> skrev:

**Comment by BroiSatse** [comment]
@Mange - I had roadie 3.1.1 and css_parser 1.3.7. 
Upgraded to roadie 3.2.0 with css_parser 1.4.6 - the issue is still present. 
What is quite strange is that `premailer-rails` seems not to have that issue, I'd think they are using the same parser?

**Comment by Mange** [comment]
Yeah, I'm pretty sure we have tests in roadie to show that this behavior
shouldn't happen.

I might have some time to look at this later during the week.

Den lör 26 nov. 2016 13:35Stanislaw Klajn <notifications@github.com> skrev:

**Comment by BroiSatse** [comment]
@Mange - I had a quick look and it seems that media queries were sort of ignored. I've implemented the changes to take them into an account when deciding given StyleBlock is inlinable or not, based on how it is handled in premailer (I am not very happy with counting brackets, but this is the way they are handling it and I assume there might be some edge cases that caused that to be that way)

**Comment by lfportal** [comment]
Any progress on this one? Would love to use @BroiSatse solution.

**Comment by alonsogarciapablo** [comment]
+1

**Comment by shmargum** [comment]
I am also seeing that media queries will end up adding both values inline only.
It would be nice if the media query could be pulled as "impossible to inline" with the :hovers and :afters as a nice progressive enhancement.

**Comment by Mange** [comment]
This should be fixed in latest `master` after #157. Please test and I'll cut a new release.
